scripts/zones/Crawlers_Nest_[S]/npcs/Rodeupansat.lua

```diff
@@ -0,0 +1,60 @@
+-----------------------------------
+-- Area: Crawlers_Nest_[S]
+--  NPC: Rodeupansat
+-- Starts and Finishes Quest: EVIL_AT_THE_INLET
+-- @zone 171
```

Comment:
```lua
-----------------------------------
-- Area: Crawlers_Nest_[S]
--  NPC: Rodeupansat
-- Starts and Finishes Quest: EVIL_AT_THE_INLET
-- !pos 129.240 -34.353 25.927 171
-----------------------------------
package.loaded["scripts/zones/Crawlers_Nest_[S]/TextIDs"] = nil;
-----------------------------------
require("scripts/zones/Crawlers_Nest_[S]/TextIDs");
require("scripts/globals/settings");
require("scripts/globals/keyitems");
require("scripts/globals/quests");
-----------------------------------
```
please. we stopped having `@zone` ages ago its just a param on `!pos` 

(I only moved the require order for consistent aesthetics)